build/Targets/Settings.props

```diff
@@ -124,6 +124,7 @@
     <When Condition="'$(MSBuildProjectExtension)' == '.vbproj' OR '$(Language)' == 'VB' OR '$(ProjectLanguage)' == 'VB'">
       <PropertyGroup>
         <ProjectLanguage>VB</ProjectLanguage>
+        <LangVersion>Latest</LangVersion>
```

Comment:
❗️ Do not use this, here or anywhere aside from unit tests that verify that it works. Upgrading the language version should always be intentional.